### PR TITLE
[MRG] Version 0.19.2

### PR DESCRIPTION
**Pull request by rth**
~~This tests the upcoming 0.19.2 release can be built without errors. Do not merge.~~

Packages the 0.19.2 release

### CODE REVIEW

**Comment by rth** [comment]
@conda-forge-admin, please rerender

**Comment by ogrisel** [comment]
There is a failure:

https://travis-ci.org/conda-forge/scikit-learn-feedstock/jobs/404161032

```
Traceback (most recent call last):
  File "/Users/travis/miniconda3/bin/conda-build", line 11, in <module>
    sys.exit(main())
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/conda_build/cli/main_build.py", line 420, in main
    execute(sys.argv[1:])
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/conda_build/cli/main_build.py", line 411, in execute
    verify=args.verify)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/conda_build/api.py", line 200, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/conda_build/build.py", line 2177, in build_tree
    notest=notest,
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/conda_build/build.py", line 1571, in build
    built_package = bundlers[output_d.get('type', 'conda')](output_d, m, env, stats)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/conda_build/build.py", line 938, in bundle_conda
    files = post_process_files(metadata, initial_files)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/conda_build/build.py", line 800, in post_process_files
    post_build(m, new_files, build_python=python)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/conda_build/post.py", line 649, in post_build
    post_process_shared_lib(m, f, prefix_files)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/conda_build/post.py", line 599, in post_process_shared_lib
    mk_relative_osx(path, m.config.host_prefix, m.config.build_prefix, files=files)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/conda_build/post.py", line 346, in mk_relative_osx
    names = macho.otool(path)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/conda_build/os_utils/macho.py", line 167, in otool
    raise CalledProcessError
TypeError: __init__() missing 2 required positional arguments: 'returncode' and 'cmd'
```

**Comment by rth** [comment]
Yes, it's due to https://github.com/conda-forge/scikit-learn-feedstock/pull/66#issuecomment-394133144 maybe I shouldn't have re-rended.

Will remove the last commit and see if it helps..

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 source:
-  url: https://github.com/scikit-learn/scikit-learn/archive/{{ version }}.tar.gz
-  sha256: 2947f531ad52e81b2436b26608d8198778ac0b4baa7d2925db8b3b3fcb39c8a2
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b276739a5f863ccacb61999a3067d0895ee291c95502929b2ae56ea1f882e888
 
 build:
-  number: 201
+  number: 0
```
Comment (jakirkham) [inline recipe/meta.yaml]: So this needs to be `200`.

ref: https://conda-forge.org/docs/blas.html

Comment (jakirkham) [reply recipe/meta.yaml]: We could also use this trick ( https://github.com/conda-forge/numpy-feedstock/commit/4a98d05bc762321aadb6a385edacb2171d53adad ).

Comment (rth) [reply recipe/meta.yaml]: Thanks for the review! Set it to 200, I think it's still more explicit that setting then incrementing the `build_number` variable..

**Comment by jakirkham** [comment]
Thanks @rth and @ogrisel :)

**Pull request merged**